src/mlpack/methods/ann/layer/add_merge_impl.hpp

```diff
@@ -150,19 +150,17 @@ template<typename InputDataType, typename OutputDataType,
          typename... CustomLayers>
 template<typename Archive>
 void AddMerge<InputDataType, OutputDataType, CustomLayers...>::serialize(
-    Archive& ar, const unsigned int version)
+    Archive& ar)
 {
   // Be sure to clear other layers before loading.
   if (Archive::is_loading::value)
     network.clear();
 
-  ar & BOOST_SERIALIZATION_NVP(network);
-  ar & BOOST_SERIALIZATION_NVP(model);
-  ar & BOOST_SERIALIZATION_NVP(run);
-
-  if (version >= 1)
-    ar & BOOST_SERIALIZATION_NVP(ownsLayers);
-  else if (Archive::is_loading::value)
+  ar & CEREAL_VECTOR_VARIANT_POINTER(network);
+  ar & CEREAL_NVP(model);
+  ar & CEREAL_NVP(run);
+  ar & CEREAL_NVP(ownsLayers);
```

Comment:
I guess we don't need to serialize `ownsLayers` if we are setting it if the Archive is loading.  I suppose we could remove this line with no ill effects.